packages/application-shell/src/components/app-bar/app-bar.tsx

```diff
@@ -83,7 +83,12 @@ const AppBar = (props: Props) => {
           </Spacings.Inline>
         </Spacings.Inline>
       </div>
-      <div>
+      <div
+        css={css`
+          display: flex;
+        `}
+      >
+        <div id="extra-menu"></div>
```

Comment:
Nit: I would define the ID here: https://github.com/commercetools/merchant-center-application-kit/blob/db6e172ce4d679a66fc6030e4f67c4fb5661d065/packages/application-shell/src/constants.ts#L13-L15